0_intro/exercises/fibonacci/src/lib.rs

```diff
@@ -1,5 +1,18 @@
 pub fn fibo(v: u32) -> u32 {
-    todo!()
+    if v == 0 {
+        0
+    }
+    else {
+        let mut f0 = 0;
+        let mut f1 = 1;
+        let mut tmp;
+        for _i in 1..v {
+            tmp = f1;
+            f1 += f0;
+            f0 = tmp
+        }
+        f1
+    }
```

Comment:
Bien! Si jamais tu fais un return tres tot (comme ici ligne 3, ou tu retournes 0 si jamais v vaut 0), c'est quand meme plus propre d'utiliser le keyword return que faire un grand else.


```suggestion
    if v == 0 {
        return 0;
    }
  
      let mut f0 = 0;
      let mut f1 = 1;
      let mut tmp;
      for _i in 1..v {
          tmp = f1;
          f1 += f0;
          f0 = tmp
      }
      f1
```